### ISSUE TITLE
Overheat

### ISSUE THREAD

**Issue by talentoscope** [RainbowMiner/RainbowMiner#1996]
I use Afterburner and have only 1 profile (number 1), and that is limited to 65c 

I woke up about half an hour ago, saw I got a kawpow rental on MRR and thatr my GPU was running at over 80c.

I'm not sure why:

A. RBM didn't kill it.
B. Why the overheat.

But this seems pretty serious to me.

Debug added.
[debug_2022-03-10.zip](https://github.com/RainbowMiner/RainbowMiner/files/8221377/debug_2022-03-10.zip)

**Comment by RainbowMiner** [comment]
You should check, if the Afterburner is configured correctly. That's the software, that should throttle your GPU's memory and core clock, if the temperature is above the limit. RainbowMiner has only one means: it can issue a notification, if the temperature reaches the value of `MinerStatusMaxTemp` (in config.txt). It will send an email and notify you over Pushover. There is currently no kill-switch implemented, that would stop a miner, if the temperature too high.
What I could do is add an extra parameter like `MinerKillMaxTemp` and a function, that would either restart or block a miner in case that temperature would be reached by a GPU.

**Comment by talentoscope** [comment]
Yeah man, I killed the miner as soon as I noticed, and re-enabled Profile 1 in afterburner.
I've not got a clue why that happened, but if it's not RBM, then that's cool.

Yeah a maxtemp killer would be awesome if you could implement that! :)
